tools/explorer-thumbnailer/src/logging.rs

```diff
@@ -0,0 +1,32 @@
+#[cfg(feature = "winlog")]

```

Comment:
Why it's optional?

Reply:
I thought I'd add it just for debugging, but if you're ok with it...